src/test/java/teammates/test/cases/ui/browsertests/AdminHomePageUiTest.java

```diff
@@ -91,7 +93,29 @@ private void testCreateInstructorAction() throws Exception {
         instructor.email = "AHPUiT.instr1@gmail.tmt";
         String institute = "TEAMMATES Test Institute 1";
         String demoCourseId = "AHPUiT.instr1.gma-demo";
-    
+        
+        ______TS("action fail: the number of columns is invalid");
+        homePage.createInstructorByInstructorDetailsForm(instructor.name + " | " + instructor.email);
+        assertEquals(homePage.getStatus(), String.format(Const.StatusMessages.INSTRUCTOR_DETAILS_LENGTH_INVALID, 
+                                                         AdminInstructorAccountAddAction.LENGTH_FOR_NAME_EMAIL_INSTITUTION)); 
+        ______TS("action success : create instructor account by Instructor Form");
```

Comment:
We have two forms in the page. So the names should identify each form uniquely. Both are instructor Details Forms, so that name does not identify either form uniquely.